### PR TITLE
Add Test preview page

### PR DESCRIPTION
**Pull request by SriramPatibanda**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Fixes #46 
- Adds Test Preview Page to the app, the user can now click on a quiz in the home screen and navigate to the test preview page.
- Removes unnecessary comments

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### Build Status
[pr-build-test](https://github.com/dscnitrourkela/project-maple/actions/workflows/pull_request.yml)

### CODE REVIEW

File: aptiche/lib/services/graphql.dart [hunk 1]
```diff
@@ -63,18 +64,28 @@ class GraphQLService {
     try {
       final QueryResult result = await _client.query(options);
       if (result.hasException) {
-        print(result.exception);
+        debugPrint(result.exception.toString());
       }
 
       /// Takes in data from [QueryResult] and converts it to a map
```
Comment (nayakastha) [inline aptiche/lib/services/graphql.dart:70]: documentation!! waah waah 🚀

**Review by abhibhaw — APPROVED** [review]

File: aptiche/lib/widgets/HomeScreen/drawer.dart [hunk 2]
```diff
@@ -33,25 +33,6 @@ class MainDrawer extends StatelessWidget {
                 softWrap: true,
                 style: Theme.of(context).primaryTextTheme.headline2,
               ),
-              /* leading: SizedBox(
```
Comment (nayakastha) [inline aptiche/lib/widgets/HomeScreen/drawer.dart:36]: Don't remove this. It's a temporary profile image generator, we have to find a substitute for [this](https://pub.dev/packages/text_drawable_flutter). 
Try using [flutter_text_drawable: ^1.0.0](https://pub.dev/packages/flutter_text_drawable)

**Review by nayakastha — CHANGES_REQUESTED** [review]

File: aptiche/lib/widgets/HomeScreen/drawer.dart (hunk 2 above)
Comment (abhibhaw) [reply aptiche/lib/widgets/HomeScreen/drawer.dart:36]: What? No more apna package? 😢

Comment (nayakastha) [reply aptiche/lib/widgets/HomeScreen/drawer.dart:36]: Mil gaya null safe wala.
Apna koi unique wala banayenge

Comment (nayakastha) [reply aptiche/lib/widgets/HomeScreen/drawer.dart:36]: merge ho gaya pr. Mujhe abhi show kiya😒

Comment (abhibhaw) [reply aptiche/lib/widgets/HomeScreen/drawer.dart:36]: Peace

Peace 🕊🕊

**Pull request merged**